airbyte-integrations/connectors/source-mysql-v2/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlJdbcPartition.kt

```diff
@@ -176,8 +176,8 @@ sealed class MysqlJdbcResumablePartition(
     open val isLowerBoundIncluded: Boolean = false
 }
 
-/** Implementation of a [JdbcPartition] for a snapshot partition. */
-class MysqlJdbcSnapshotPartition(
+/** RFR for cursor based read. */
+class MysqlJdbcRfrSnapshotPartition(
```

Comment:
All mysql snapshot partition are going to be MysqlJdbcRfrSnapshotPartition?
This may be confusing (or I misread)

I think logically we have MysqlJdbcSnapshotPartition and RFR specialization which does the same thing but emits a different final state?